resources/dashboard.rb

```diff
@@ -19,6 +19,11 @@
 # limitations under the License.
 #
 
+unified_mode true
+
+deprecated 'The `grafana_dashboard` resources will be removed in the next major version of the grafana cookbook. ' \
+  'Grafana 8.0 removed support for the dashboard API these use. Use e.g. Terraform to manage dashboards instead.'
```

Comment:
Odd, I'm not seeing this deprecation when I run it.

Reply:
I've had a quick poke around and I'm not entirely unconvinced that they aren't broken at the minute, they definitely did used to work. From what I saw the `deprecated` method in `chef_class.rb` (assuming that's the correct one) doesn't seem to be getting hit for whatever reason. I've not had time to dig into it any further yet.

Reply:
@lamont-granquist :point_up: